package.json

```diff
@@ -9,6 +9,7 @@
     "atom": ">0.50.0"
   },
   "scripts": {
+    "format": "prettier --write .",
     "lint.fix": "eslint . --fix",
```

Comment:
Why do we need `eslint --fix` and `prettier`? shouldn't we just use one or the other?

Reply:
- Eslint config only focuses on the issues of the code. 
- Prettier only formats the code (whitespace and punctuation). 

They should be used in conjunction with each other. Some people hack Prettier, so it gets integrated into Eslint, but I don't like that. I prefer to use Eslint for the actual issues of the code rather than formatting. The separation of the concerns helps to focus on different problems.

Reply:
We should test prettier in CI otherwise PRS could slip through that haven't run prettier.

Reply:
We can. Let me merge this now, and I will add the CI step for all the repos later.

Reply:
Why do you do that? You know that it is going to get forgotten. it would be easier to do it now rather than try to remember to do it later.

Reply:
It is done. Since I am doing it for all the repos, it is better to keep things consistent between the repos. 

See: https://github.com/atom-community/autocomplete-paths/commit/adb64498c73310bd72ad35005c6291fdeb744216

Reply:
Why was that any easier to do after this PR was submitted rather than being included in this PR?

Reply:
Because I have a directory that includes all the repositories (like a mono repo), and when I want to apply a change to all the packages, I use that (sometimes I write scripts). 

This is the downside of having separate repos instead of a mono repo. Keeping the repositories in sync with each other is harder.

Reply:
Why couldn't you write the script to include the change with this PR?

Reply:
> Keeping the repositories in sync with each other is harder.

Ya but they are so much easier to update when you don't have to worry about breaking some unrelated package.